.gitignore

```diff
@@ -1,8 +1,6 @@
 # User specific & automatically generated files #
 #################################################
 /composer.lock
-/phpunit.xml
```

Comment:
Shouldnt we keep the phpunit one as dist?

Reply:
I think it's easier for people to just modify phpunit.xml instead of having a possibly conflicting file overriding.

Reply:
I think the idea is to have the phpunit.xml as gitignored copy locally without the danger of accidentally committing your passwords or more into the PR.
Personally, I would like to see that kept here.

Reply:
Sure.